activestorage/lib/active_storage/transformers/image_magick.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  module Transformers
+    class ImageMagick < ImageProcessingTransformer
+      private
+        def processor
+          ImageProcessing::MiniMagick
+        end
+
+        def validate_transformation(name, argument)
+          method_name = name.to_s.tr("-", "_")
+
+          unless ActiveStorage.supported_image_processing_methods.any? { |method| method_name == method }
```

Comment:
I know you just moved it over, but I wonder why this isn't just:

```suggestion
          unless ActiveStorage.supported_image_processing_methods.include?(method_name)
```